frameworks/g-cloud-9/questions/services/multiqUserAuthenticationSoftware.yml

```diff
@@ -0,0 +1,13 @@
+name: User authentication
+question: User authentication
+
+depends:
+  - "on": lot
+    being:
+      - cloud-software
+
+type: multiquestion
+questions:
+  - userAuthenticationNeeded
+  - userAuthentication
```

Comment:
Not sure if it works, but interested to find out! Feels like it /should/, but could be wrong.